### PR TITLE
qbd direct integration advanced settings html changes

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

**Comment by DhaaraniCIT** [comment]
Screenshot 2024-09-12 at 10 52 48 AM
Screenshot 2024-09-12 at 10 53 07 AM

File: src/app/integrations/qbd-direct/qbd-direct-shared/qbd-direct-advanced-settings/qbd-direct-advanced-settings.component.html [hunk 1]
```diff
@@ -1 +1,123 @@
...
+                        [formControllerName]="'exportSchedule'">
+                    </app-configuration-toggle-field>
+                </div>
+                <div *ngIf="advancedSettingsForm.value.exportSchedule">
+                    <div class="tw-mt-12-px tw-bg-white tw-border tw-border-solid tw-border-separator tw-rounded-12-px">
+                        <app-configuration-select-field
+                        [form]="advancedSettingsForm"
+                        [isFieldMandatory]="true"
+                        [mandatoryErrorListName]="'Frequency'"
+                        [label]="'How frequently do you want the IIF file to be generated?'"
```
Comment (ashwin1111) [inline src/app/integrations/qbd-direct/qbd-direct-shared/qbd-direct-advanced-settings/qbd-direct-advanced-settings.component.html:30]: all content should come from TS file

File: src/app/integrations/qbd-direct/qbd-direct-shared/qbd-direct-advanced-settings/qbd-direct-advanced-settings.component.html [hunk 2]
```diff
@@ -1 +1,123 @@
...
+                        [subLabel]="'You can select the frequency you want your IIF file to be sent via email.'"
+                        [options]="frequencyOption"
+                        [iconPath]="'arrow-rotate'"
+                        [placeholder]="'Select frequency'"
+                        [formControllerName]="'frequency'"></app-configuration-select-field>
+    
+                        <div *ngIf="advancedSettingsForm.value.frequency">
+                            <div class="tw-pl-22-px">
+                                <div *ngIf="advancedSettingsForm.value.frequency === QBDScheduleFrequency.DAILY" >
```
Comment (ashwin1111) [inline src/app/integrations/qbd-direct/qbd-direct-shared/qbd-direct-advanced-settings/qbd-direct-advanced-settings.component.html:39]: design would be similar to qbo

**Comment by DhaaraniCIT** [comment]
Comments are fixed in https://github.com/fylein/fyle-integrations-app/pull/947

**Comment by ashwin1111** [comment]
> Comments are fixed in #947

don't see any fixes around advanced settings in that PR

**Comment by DhaaraniCIT** [comment]
sorry @ashwin1111 wrong PR

**Comment by DhaaraniCIT** [comment]
Screenshot 2024-09-24 at 5 12 54 PM
Screenshot 2024-09-24 at 5 13 09 PM
Screenshot 2024-09-24 at 5 13 16 PM

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**